microsoft/testsuites/performance/storageperf.py

```diff
@@ -175,6 +175,42 @@ def perf_premium_datadisks_4k(self, node: Node, result: TestResult) -> None:
     def perf_premium_datadisks_1024k(self, node: Node, result: TestResult) -> None:
         self._perf_premium_datadisks(node, result, block_size=1024)
 
+    @TestCaseMetadata(
+        description="""
+        This test case uses fio to test data disk performance with 4K block size.
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+        requirement=simple_requirement(
+            disk=schema.DiskOptionSettings(
+                data_disk_type=schema.DiskType.StandardSSDLRS,
+                os_disk_type=schema.DiskType.StandardSSDLRS,
+                data_disk_iops=search_space.IntRange(min=6000),
+                data_disk_count=search_space.IntRange(min=16),
+            ),
+        ),
+    )
+    def perf_standardssd_datadisks_4k(self, node: Node, result: TestResult) -> None:
+        self._perf_premium_datadisks(node, result)
```

Comment:
please 
add standardssd for DiskType and specify the disk_type=DiskType.standardssd here